<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clustering</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gb_in">
      <property name="title">
       <string>Inputs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_path_lmap">
         <property name="text">
          <string>Label map path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_path_lmap">
         <property name="text">
          <string>../../examples/analyze/out/tomo9_bin1_labelmap.mrc</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_cradius">
         <property name="toolTip">
          <string>Should correspond to average radius of target objects (in voxels)</string>
         </property>
         <property name="text">
          <string>Cluster radius</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_cradius">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_sthr">
         <property name="toolTip">
          <string>Found objects smaller than this threshold are discarded</string>
         </property>
         <property name="text">
          <string>Size threshold</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_csize_thr">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gp_out">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_path_objl">
         <property name="text">
          <string>Object list path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_path_objl">
         <property name="text">
          <string>out/objlist.xml</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_launch">
      <property name="text">
       <string>Launch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="te_terminal_out"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
